src/mirror.c

```diff
@@ -402,7 +405,16 @@ enum swupd_code mirror_main(int argc, char **argv)
 	}
 
 	/* print new configuration */
-	print_update_conf_info();
+	if (quiet) {
```

Comment:
why not just use `if (log_is_quiet())` here?

Reply:
🤔 good point 😆